licenses/DUMB-0.9.3

```diff
@@ -0,0 +1,87 @@
+ * If you do not agree to these terms, please        |  \/ /\   /
+ * do not use DUMB.                                   \_  /  > /
+ *                                                      | \ / /
+ * Information in [brackets] is provided to aid         |  ' /
+ * interpretation of the licence.                        \__/
+ */
+
+
+Dynamic Universal Music Bibliotheque, Version 0.9.3
+
+Copyright (C) 2001-2005 Ben Davis, Robert J Ohannessian and Julien Cugniere
+
+This software is provided 'as-is', without any express or implied warranty.
+In no event shall the authors be held liable for any damages arising from the
+use of this software.
+
+Permission is granted to anyone to use this software for any purpose,
+including commercial applications, and to alter it and redistribute it
+freely, subject to the following restrictions:
+
+1. The origin of this software must not be misrepresented; you must not claim
+   that you wrote the original software. If you use this software in a
+   product, you are requested to acknowledge its use in the product
+   documentation, along with details on where to get an unmodified version of
+   this software, but this is not a strict requirement.
+
+   [Note that the above point asks for a link to DUMB, not just a mention.
+   Googling for DUMB doesn't help much! The URL is "http://dumb.sf.net/".]
+
+   [The link was originally strictly required. This was changed for two
+   reasons. Firstly, if many projects request an acknowledgement, the list of
+   acknowledgements can become quite unmanageable. Secondly, DUMB was placing
+   a restriction on the code using it, preventing people from using the GNU
+   General Public Licence which disallows any such restrictions. See
+   http://www.gnu.org/philosophy/bsd.html for more information on this
+   subject. However, if DUMB plays a significant part in your project, we do
+   urge you to acknowledge its use.]
+
+2. Altered source versions must be plainly marked as such, and must not be
+   misrepresented as being the original software.
+
+3. This notice may not be removed from or altered in any source distribution.
+
+4. If you are using the Program in someone else's bedroom on any Monday at
+   3:05 pm, you are not allowed to modify the Program for ten minutes. [This
+   clause provided by Inphernic; every licence should contain at least one
+   clause, the reasoning behind which is far from obvious.]
+
+5. Users who wish to use DUMB for the specific purpose of playing music are
+   required to feed their dog on every full moon (if deemed appropriate).
+   [This clause provided by Allefant, who couldn't remember what Inphernic's
+   clause was.]
+
+6. No clause in this licence shall prevent this software from being depended
+   upon by a product licensed under the GNU General Public Licence. If such a
+   clause is deemed to exist, Debian, then it shall be respected in spirit as
+   far as possible and all other clauses shall continue to apply in full
+   force.
+
```

Comment:
Looks like clause 7 is missing?

Reply:
Clause 7 doesn't exist. It looks like it was a tongue-in-cheek prank to purposefully skip clause 7 when introducing clause 8, in order to annoy the Debian license team.